src/views/colours.njk

```diff
@@ -1,5 +1,6 @@
 ---
 title: Colours
+tags: styles
```

Comment:
Do let me know if there's a better way of distinguishing between nav categories

Reply:
I created a "collection" for styles in this PR

https://github.com/ONSdigital/dp-design-system/pull/9/files#diff-c306e0a99961a16f5c5c83996caa0958b94006d97f97475049ea3a08036bb5b0R13

Reply:
Thanks. Will implement as such